README.md

```diff
@@ -1 +1,140 @@
-# Leaderboard-
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-Leaderboard)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#amen-and-Isma)
+- [🔭 Future Features](#the other pages)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#microverse)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Awesome Books] <a name="Leaderboard"></a>
+
+**[Leaderboard]** This project is a web application that displays a leaderboard of top players and their scores in a game. The application retrieves data from an API and dynamically displays the leaderboard in a user-friendly format on a web page.
+
+## 🛠 Built With <a name="HTML, CSS and JAVASCRIPT"></a>
+
+### Tech Stack <a name="HTML, CSS and JAVASCRIPT"></a>
+
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Es6-Modules]**
+- **[API-calls]**
+- **[Recent-score-board]**
+- **[Add-score-form]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://ismailmunyentwari9.github.io/Leaderboard-/dist)
+
```

Comment:
- [ ] Please your readme looks awesome :+1:  but your live demo link is broken link so make sure you fix it. :pray: 
  ![image](https://user-images.githubusercontent.com/92303891/223489344-4a1e7c86-70c3-40ac-92e5-153e143ce3e0.png)